test/test-operation.ts

```diff
@@ -0,0 +1,11 @@
+import { CustomOperator, quote } from '../src';
+
+export class TestOperation implements CustomOperator {
```

Comment:
This should be called `TestOperator`.

Reply:
Fixed in the next commit, then updated the README to read properly as well.